helm/alfresco-content-services/values.yaml

```diff
@@ -190,8 +210,8 @@ tika:
 filestore:
   replicaCount: 1
   image:
-    repository: alfresco/alfresco-shared-file-store
-    tag: "0.1"
+    repository: quay.io/alfresco/alfresco-shared-file-store
+    tag: "ATS-94-1.0-SNAPSHOT"
```

Comment:
No snapshots, please.

Reply:
Yes, agreed. This is known to be WIP (for early review) until we resolve REPO-3749 (and ATS-94).

Reply:
Publish images to Docker Hub as -EA e.g. ATS-94-1.0-EA1 and then we can use those.
Since the code is in the Enterprise domain, and the images are EA, we can pull them or replace them as required later.